config/routes.rb

```diff
@@ -3,6 +3,9 @@
 
   # Defines the root path route ("/")
   resources :users, only: [:show, :index] do 
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show, :new, :create] do
+      resources :comments, only: [:create, :new]
+      resources :likes, only: [:create]
```

Comment:
- [ ] An error occurs when trying to reach the new comments endpoint which shows the new comment form. This is a requirement just like the previous reviewer stated. Kindly implement it 👍 
```ruby
GET   /posts/:post_id/comments/new(.:format)     comments#new
```
![image](https://user-images.githubusercontent.com/46839548/192070645-bddb6061-05f8-4a00-8671-b91c9b1f8647.png)